Firestore/core/test/unit/local/local_store_test.cc

```diff
@@ -104,6 +107,13 @@ std::vector<Document> DocMapToArray(const DocumentMap& docs) {
   }
   return result;
 }
+MaybeDocumentMap DocArrayToMap(const std::vector<MaybeDocument>& docs) {
```

Comment:
Nit: perhaps s/Array/Vector/? Otherwise, it sounds as if this function were taking a built-in array (or perhaps a `std::array`).